packages/googlehealthcare/configuration-schema.json

```diff
@@ -18,23 +18,42 @@
       "type": "string",
       "description": "The API version",
       "default": "v1",
-      "examples": ["v1", "v1beta1"]
+      "examples": [
+        "v1",
+        "v1beta1"
+      ]
     },
     "cloudRegion": {
       "type": "string",
-      "examples": ["us-central1"]
+      "title": "cloudRegion",
+      "description": " The dataset location",
```

Comment:
Is this description correct? Is it not "The region of your Google Cloud project" ?

Reply:
@josephjclark I guess we can say that, I was following this description from Google cloud docs
<img width="659" alt="Screenshot 2023-07-07 at 1 52 27 PM" src="https://github.com/OpenFn/adaptors/assets/6592749/ccecc065-c45c-4866-8ca6-ad66a926f51f">

Reply:
@josephjclark mtuchi and I confirmed that with google healthcare API, you can set a region at the dataset level, so this description does make sense.